src/main.js

```diff
@@ -139,23 +139,16 @@ backToMainButton.addEventListener("click", toggleToHome)
 
 savePosterButton.addEventListener("click", saveMainPoster)
 
-
 makePosterButton.addEventListener("click", () => {
   event.preventDefault();
-  generatePoster();
+  generatePoster(posterImgUrl.value, motivationalTitle.value, motivationalQuote.value);
   storePosterInfo();
   toggleToMain();
```

Comment:
Have you thought about creating a helper function to do the stuff that's happening in storePosterInfo? You're really doing the same thing with all three values, so maybe a helper function could DRY that up. (github won't let me comment down there for some stooopid reason 🙄)